multivariate_grouped_data_plot.Rmd

```diff
@@ -0,0 +1,825 @@
+# How to plot grouped data for multivariate (in Chinese)
+
+Yihui Hu, Liuxin Chen
+
+原始网页 Original website：http://www.sthda.com/english/articles/32-r-graphics-essentials/132-plot-grouped-data-box-plot-bar-plot-and-more/#grouped-continuous-variables
+
+```{r, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+
+
+## 1 数据准备
+
+加载所需的软件包并将主题函数theme_pubclean()设置为默认主题：
+
+```{r}
+library(dplyr) 
+library(ggplot2)
+library(ggpubr)
+theme_set(theme_pubclean())
+```
+
+  
+  
+
+## 2 分类变量分组制图
+
+绘图类型：分类变量频率的分组条形图。
+
+关键函数：geom_bar()。
+
+演示数据集：dimonds[ggplot2]。 
+
+演示示例中使用的类别变量为：
+
+- cut：切工的钻石质量（一般，良好，非常好，优质，理想）。
+
+- color：钻石色，从J（最差）到D（最佳）。
+
+在演示示例中，我们将仅绘制数据的子集（颜色J和D）。 
+
+步骤如下：
+
+a.过滤数据以仅保留颜色在其中的菱形（“ J”，“ D”）
+
+b.按切割质量和钻石颜色对数据进行分组
+
+c.按组计算频率
+
+d.创建条形图
+
+### 2.1 筛选特定分组数据及计算频率
+
+```{r}
+df <- diamonds %>%
+  filter(color %in% c("J", "D")) %>%
+  group_by(cut, color) %>%
+  summarise(counts = n()) 
+head(df, 4)
+```
+
+### 2.2 分组频率柱状图
```

Comment:
Hashtag headers should not be followed by numbers since the hashtags will create numbered subheadings.